app/src/main/java/com/app/apptodo/apptodo/list/TodoListFragment.kt

```diff
@@ -28,26 +31,24 @@ class FragmentTask: Fragment(), AppTodoContract.View {
             false
         )
 
-        return binding.root
+        val view = binding.root
+        return view
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        adaptor = AppTodoAdaptor(requireContext(), mutableListOf())
+        adaptor = TaskAdapter(requireContext(), mutableListOf())
```

Comment:
Corrigir nomenclatura
```suggestion
        adapter = TaskAdapter(requireContext())
```